build.sbt

```diff
@@ -0,0 +1,26 @@
+import sbt.Keys._
+import sbt._
+import uk.gov.hmrc.versioning.SbtGitVersioning
+import play.sbt.PlayImport.filters
+import play.core.PlayVersion
+
+val appName = "json-encryption"
+
+val dependencies = Seq(
+  "com.typesafe.play" %% "play-json" % PlayVersion.current % "provided",
```

Comment:
"2.5.12" would be lighter than `PlayVersion.current`

Reply:
Ok, then I'll revert that change. It seems it doesn't provide the results I expected.